packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -85,16 +78,11 @@ exports[`Android API levels Android API>=21 should render with ripple 1`] = `
 <View
   style={
     Array [
-      Array [
-        Object {
-          "borderRadius": 40,
-          "overflow": "hidden",
-        },
-      ],
-      null,
       Object {
-        "backgroundColor": "rgb(0, 215, 117)",
```

Comment:
Any reason is not set?

Reply:
Same answer as this pretty much: https://github.com/Skyscanner/backpack-react-native/pull/117#discussion_r279404610